openhands/agenthub/codeact_agent/tools/str_replace_editor.py

```diff
@@ -1,11 +1,13 @@
 from litellm import ChatCompletionToolParam, ChatCompletionToolParamFunctionChunk
 
-_STR_REPLACE_EDITOR_DESCRIPTION = """Custom editing tool for viewing, creating and editing files in plain-text format
+_STR_REPLACE_EDITOR_DESCRIPTION = """Custom editing tool for viewing, creating and editing files.
 * State is persistent across command calls and discussions with the user
-* If `path` is a file, `view` displays the result of applying `cat -n`. If `path` is a directory, `view` lists non-hidden files and directories up to 2 levels deep
+* If `path` is a file, `view` displays the file content in Markdown format. If `path` is a directory, `view` lists non-hidden files and directories up to 2 levels deep
+* The following file extensions can be viewed in Markdown format: [".xlsx", ".pptx", ".wav", ".mp3", ".m4a", ".flac", ".pdf", ".docx", ".csv", ".jsonld", ".html", ".htm", ".txt"], and all other types of text files. IT DOES NOT HANDLE IMAGES.
```

Comment:
I don't understand why xlsx/pptx/wav/mp3/m4a/pdf... can be viewed in Markdown format.

Reply:
Oh I see, you changed openhands-aci tool